src/js/actions/menu.js

```diff
@@ -75,14 +75,18 @@ define(function (require, exports) {
         }
 
         var isPlaceCommand = payload.commandID === _PLACE_LINKED_MENU_ID ||
-                payload.commandID === _PLACE_EMBEDDED_MENU_ID;
+            payload.commandID === _PLACE_EMBEDDED_MENU_ID;
         
         return Promise.bind(this)
             .then(function () {
+                // This is a hack for the place linked/embedded menu commands which do not
+                // seem to promptly emit a toolModalStateChanged:enter event
```

Comment:
Created a non-M3 watson bug for this issue (not getting toolModalStateChanged).  We'll stick with this workaround for now.   https://watsonexp.corp.adobe.com/#bug=4074688